apps/files/src/components/BreadCrumbs.vue

```diff
@@ -171,8 +172,10 @@ export default defineComponent({
 				}
 			}
 			if (node === undefined) {
+				const view = this.views.find(view => view.params?.dir === dir)
 				return {
 					...this.$route,
+					params: { fileid: String(view?.params?.fileid) },
```

Comment:
If this is undefined then fileid will be the string `undefined`, no? So maybe better:
```suggestion
					params: { fileid: String(view?.params?.fileid ?? '') },
```